pkg/estimator/client/accurate.go

```diff
@@ -127,22 +127,18 @@ func getClusterReplicasConcurrently(parentCtx context.Context, clusters []string
 	ctx, cancel := context.WithTimeout(parentCtx, timeout)
 	defer cancel()
 
-	availableTargetClusters := make([]workv1alpha2.TargetCluster, len(clusters))
-
-	var wg sync.WaitGroup
-	errChan := make(chan error, len(clusters))
-	for i := range clusters {
-		wg.Add(1)
-		go func(idx int, cluster string) {
-			defer wg.Done()
+	clusterReplicas := make([]workv1alpha2.TargetCluster, len(clusters))
+	funcs := make([]func() error, len(clusters))
+	for i, cluster := range clusters {
+		i, cluster := i, cluster
```

Comment:
I'd like to change the name in order to prevent confusion.

Reply:
Rename the variables `i, cluster` in inner loop may be better.

Reply:
@Garrybest updated